src/button/template/containerTemplate.jsx

```diff
@@ -64,6 +64,10 @@ function getDimensions({ label, size, tagline, fundingicons, layout, number, vie
         height = (buttonHeight * number) + (perc(buttonHeight, BUTTON_RELATIVE_STYLE.VERTICAL_MARGIN) * (number - 1));
     }
 
+    if (layout === BUTTON_LAYOUT.VERTICAL && (size === BUTTON_SIZE.LARGE || size === BUTTON_SIZE.MEDIUM)) {
```

Comment:
Shouldn't we check `cards.length` here so the check is the same as in the template?

Reply:
what's a template you're talking about? the `xo-button` one?

Reply:
I mean here: https://github.com/paypal/paypal-checkout/pull/758/files#diff-4957c6fdc3594ef3bb401ab5307f46d2R349